SECURITY.md

```diff
@@ -256,6 +256,14 @@ embargo policy to share binary distributions of the security fixes before the pu
 This includes, but is not limited to, Envoy binaries and Docker images. It is expected that
 distributors have a method to stage and validate new binaries without exposing them publicly.
 
+If Envoy is running in a hosted environment, the fix may rollout ahead of the embargo lift date
+only if the end user **cannot** access the running Envoy process and possibly reverse engineer the fix.
+If a security fix includes changes to the controlplane then the controlplane binaries, Docker images,
```

Comment:
Same comment as above. Break them into list of explicit clauses so it's easier to parse.